tests/autofix_files/noqa_testing.py

```diff
@@ -1,5 +1,4 @@
 # AUTOFIX
-# NOANYIO # TODO
```

Comment:
lots of the eval files marked with `# NOANYIO` were in fact fully working running against anyio.